test/static/UserData/user_data.sh

```diff
@@ -14,9 +14,9 @@ BUCKET_ROOT_DIR=
 TARGET_UTIL=0
 
```

Comment:
`set -euo pipefail`

Reply:
same as response for `setup.go` 

I'll address this in next revision